s3_prototype/main.tf

```diff
@@ -65,7 +65,8 @@ data "aws_iam_policy_document" "policy_document" {
 }
 
 resource "aws_s3_bucket_policy" "bucket_policy" {
-  bucket     = aws_s3_bucket.bucket.*.id[0]
```

Comment:
this * [0] syntax was unnecessary as we just have one `aws_s3_bucket.bucket` in the module.